src/campaigns/tasks/fixtures/tasks-week-1.ts

```diff
@@ -47,6 +51,7 @@ const tasksWeek1: CampaignTask[] = [
     cta: 'Write Post',
     week: 1,
     flowType: CampaignTaskType.socialMedia,
+    defaultAiTemplateId: 'GpWsRql46Nif2wYroxj81',
```

Comment:
This is gonna be very brittle given that Contentful users can delete these on a whim 😕